Makefile

```diff
@@ -27,7 +27,9 @@ all: format generate build test
 build: operator prometheus-config-reloader
 
 operator: $(GOLANG_FILES)
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o $@ cmd/operator/main.go
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build \
+	-ldflags "-X github.com/coreos/prometheus-operator/pkg/version.Version=$(shell cat VERSION)" \
```

Comment:
I think it would be a good idea to do the same for the config reloader.

Reply:
Do I understand correctly, that you want to release the config-reloader along with the Prometheus Operator and the Prometheus Config Reloader? Is that necessary, given the fact, that it never changed so far?

Reply:
Sorry I meant that it would be a good idea for the prometheus-config-reloader as well, so we can tell the version from the logs people share with us.

Reply:
Good idea. Adjusted accordingly.